internal/mithril-persistence/src/database/repository/cardano_transaction_repository.rs

```diff
@@ -1215,10 +1234,161 @@ mod tests {
             .unwrap();
 
         repository
-            .remove_rolled_back_transactions_and_block_range(BlockRange::LENGTH * 3)
+            .remove_rolled_back_transactions_and_block_range_by_block_number(BlockRange::LENGTH * 3)
             .await
             .unwrap();
         assert_eq!(2, repository.get_all_transactions().await.unwrap().len());
         assert_eq!(2, repository.get_all_block_range_root().unwrap().len());
     }
+
+    #[tokio::test]
+    async fn remove_rolled_back_transactions_and_block_range_by_slot_number() {
```

Comment:
Is it `remove_rolled_back_transactions_and_block_range_by_block_number` or `remove_rolled_back_transactions_and_block_range_by_slot_number` we want to test here ?
Test name does not match with function called.

Reply:
Fixed, thanks for pointing that out :+1: